app/src/utils/FakeData.ts

```diff
@@ -0,0 +1,3102 @@
+export const diabetes = [
```

Comment:
Pinged you about this as well, but I thought we were expecting CSV? Just wanted to clarify since I thought we told Brian we wanted CSV and not JSON.

Reply:
Per offline discussion - I don't have a strong preference for csv vs json, but I think it's easier to work with json, as long as it's in a flattened format where it's structured as an array of rows, and each row is key/value pairs.